web/src/main/resources/static/js/main.js

```diff
@@ -80,28 +80,57 @@ $(document).ready(function(){
       sorted_version_list.forEach(
         function(item) {
           var _hash = '#/home?version=' + item;
-          var selected_attribute = '';
+          var selected_class = '';
           if (displayed_version === item) {
-            selected_attribute = ' selected';
+            selected_class = ' active';
           }
-          var option = '<option data-desc="' + available_versions[item].description + '" class="item" hash="' + _hash + '" ' + selected_attribute + '>' + item + '</option>';
+          var option = '<li><a href="#" class="dropdown-item' + selected_class + '" data-desc="' + available_versions[item].description + '"  hash="' + _hash + '">' + item + '</a></li>';
           var this_item_is_visible = available_versions[item].visible;
           if (!this_item_is_visible && previous_version_was_visible) {
-            option_list_html.push('<option class="item" hash="disabled" disabled>' + '&HorizontalLine;'.repeat(10) + '</option>')
+            option_list_html.push('<li class="dropdown-submenu"><a class="dropdown-item dropdown-toggle" href="#">More</a><ul class="dropdown-menu">');
```

Comment:
I would label the button with an indication that this button will go somewhere .. like "More..." or "Show more choices" or maybe use an icon/glyph to indicate that it will open (like a down-facing triangle)

Reply:
@sheridancbio the class dropdown-toggle does give the button an arrow to show that it has a submenu.

Reply:
👍